typescript/infra/src/xerc20/utils.ts

```diff
@@ -174,6 +174,39 @@ export async function updateChainLimits({
     bufferCap: currentBufferCap,
   } = await xERC20Adapter.getRateLimits(bridgeAddress);
 
+  // if buffer cap is 0, remove the bridge
+  if (bufferCap === 0) {
+    // return if the bridge is already removed
+    if (currentBufferCap === BigInt(0)) {
+      rootLogger.info(
+        chalk.yellow(`[${chain}][${bridgeAddress}] Bridge already removed`),
+      );
+      return;
+    }
+
+    const removeBridgeTx = await prepareRemoveBridgeTx(
+      chain,
+      xERC20Adapter,
+      bridgeAddress,
+    );
+    if (!dryRun) {
+      await sendTransactions(
+        envMultiProvider,
+        chain,
+        [removeBridgeTx],
+        xERC20Address,
+        bridgeAddress,
+      );
+    } else {
+      rootLogger.info(
+        chalk.gray(
+          `[${chain}][${bridgeAddress}] Dry run, no transactions sent, exiting...`,
+        ),
+      );
+    }
+    return;
```

Comment:
i suggest to use early return and avoid use negative logic (it is just easier to read)


```suggestion

    if (dryRun) {
      rootLogger.info(
        chalk.gray(
          `[${chain}][${bridgeAddress}] Dry run, no transactions sent, exiting...`,
        ),
      );
      return;
    }

    await sendTransactions(
        envMultiProvider,
        chain,
        [removeBridgeTx],
        xERC20Address,
        bridgeAddress,
      );
```

something in that direction